test/IntegrationTests/SmokeTests.cs

```diff
@@ -295,6 +295,38 @@ public void LogsNoneInstrumentations()
 
         collector.AssertEmpty(5.Seconds());
     }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void TracesNoneInstrumentations()
+    {
+        using var collector = new MockSpansCollector(Output);
+        SetEnvironmentVariable("OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS", "none");
+        RunTestApplication();
+        collector.AssertEmpty(5.Seconds());
+    }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void MetricsNoneInstrumentations()
+    {
+        using var collector = new MockMetricsCollector(Output);
+        SetEnvironmentVariable("OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS", "none");
+        RunTestApplication();
+        collector.AssertEpmty();
+    }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void  AutoLogsDisabledInstrumentationsForILogger()
```

Comment:
How about renaming to

```suggestion
    public void LogsDisabledInstrumentation()
```

?